src/collects/seashell/backend/dispatch.rkt

```diff
@@ -451,7 +451,7 @@
         (_ _) ...)
        (define checksum
          (write-file project file (string->bytes/utf-8 contents)
-                     (if history (string->bytes/utf-8 history) "")
+                     (if history history "")
```

Comment:
I had changed it to a string to get it to work with json->string (or string->json?) somewhere outside my code. Is there somewhere specific I can change it to a string and still leave it as bytes everywhere else?